README.md

```diff
@@ -34,13 +34,25 @@ Load and modify an [official example OME-Zarr](https://zenodo.org/record/7274533
 import numpy as np
 from iohub.ngff import open_ome_zarr
 
-with open_ome_zarr("20200812-CardiomyocyteDifferentiation14-Cycle1.zarr") as dataset:
+with open_ome_zarr("test.zarr", mode="r", layout="auto") as dataset:
```

Comment:
I actually don't mind the previous idea of having the specific dataset name along with the link directly above it.

Reply:
I'm open to other solutions, though. Maybe there's a faster way of getting a small `test.zarr` dataset in the user's working directory.

Reply:
Yes the original file name is better. Reverting